resources/maskinporten_clients.py

```diff
@@ -167,6 +178,12 @@ def create_client_key(
         client_id=client_id,
     )
 
+    create_okdata_permissions(
+        resource_name=f"okdata:maskinporten-key:{kid}",
```

Comment:
I så fall blir det `{env}-{client_id}-key-{kid}` i stedet for `{kid}` på slutten her.